browser/html/cool.html.m4

```diff
@@ -247,7 +247,7 @@ m4_ifelse(MOBILEAPP, [true],
       data-enable-debug = "%ENABLE_DEBUG%"
       data-frame-ancestors = "%FRAME_ANCESTORS%"
       data-socket-proxy = "%SOCKET_PROXY%"
-      data-ui-defaults = "%UI_DEFAULTS%"
+      data-ui-defaults = '%UI_DEFAULTS%'
```

Comment:
I guess this does the trick, but a safer way could be to use base64 encoding of the json string and decode that in the global.js then which could help prevent future issues in case the UI defaults json contains any single quote

Reply:
That's right, though it's hard to guess if we need single quote in UI defaults where almost everything is either true, false or one word.

Reply:
Let me update and thanks for the comment :)